### PR TITLE
Added support for calexp cutout images by id

### PR DESCRIPTION
**Pull request by kennywlo**
Changed implemented per SUIT requirement.

### CODE REVIEW

File: python/lsst/dax/imgserv/imageREST_v0.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/b:in/env python
```
Comment (jgates108) [inline python/lsst/dax/imgserv/imageREST_v0.py]: ':' snuck in there

File: python/lsst/dax/imgserv/imageREST_v0.py [hunk 2]
```diff
@@ -290,11 +332,50 @@ def _getICutout(_request, W13db, units):
...
+    img.writeFits(fileName)
+    resp = responseFile(fileName)
+    os.remove(fileName)
+    os.removedirs(tmpPath)
+    return resp
 
+
+def _getICutoutFromScienceId(_request, W13db, scienceId):
+    '''Get cutoff of calexp image from the id given.
+    W13db presumed to be W13CalexpDb.
```
Comment (jgates108) [inline python/lsst/dax/imgserv/imageREST_v0.py]: 'cutout' instead of 'cutoff'

File: python/lsst/dax/imgserv/locateImage.py [hunk 3]
```diff
@@ -450,9 +470,38 @@ def _cutoutBoxPixels(srcImage, xyCenter, width, height, log):
...
+
+def _getKeysFromList(flist, fields):
+    '''flist assumed to be dictionary
+    '''
+    vals = []
+    for f in fields:
+        for l in flist:
+            if f == l:
+                vals.append(flist.get(f))
+    return vals
```
Comment (jgates108) [inline python/lsst/dax/imgserv/locateImage.py]: 'l' should not be used for a variable name. It looks too much like '1'.

Would this work?
```
for f in fields:
    vals.append(flist.get(f))
```

File: python/lsst/dax/imgserv/imageREST_v0.py (hunk 1 above)
Comment (kennywlo) [inline python/lsst/dax/imgserv/imageREST_v0.py]: fixed

File: python/lsst/dax/imgserv/imageREST_v0.py (hunk 2 above)
Comment (kennywlo) [inline python/lsst/dax/imgserv/imageREST_v0.py]: changed.

File: python/lsst/dax/imgserv/locateImage.py (hunk 3 above)
Comment (kennywlo) [inline python/lsst/dax/imgserv/locateImage.py]: Will test with this suggested change, to be kept if no issue.

Comment (jgates108) [inline python/lsst/dax/imgserv/locateImage.py]: or 
```
for f in fields:
    elem = flist.get(f)
    if elem is not None:
        vals.append(elem);
```

**Pull request merged**